src/main/java/api/Message.java

```diff
@@ -5,4 +5,8 @@
 public class Message {
     @JsonProperty("message")
     public String message;
+
+    public Message(String message) {
```

Comment:
This is what it was named in the Dropwizard tutorial and I kept it, but I can see how it would be confusing. What do you think about the name `helpers` or `templates`? maybe `services`?

Reply:
package should represent the code usage.  Currently you've got both data objects and the ExceptionHandler which makes sense if it was api but may not if separated differently.  Typically how you want to think of packages is by grouping things with a similar functional use together.  Resources are a good example since they all deal with the api layer (if you had multiple)